### PR TITLE
Update README.md to clarify how it can be used

### PR DESCRIPTION
**Pull request by designium**
It is not clear from the readme.md that the solidity-coverage has to be install per project basis and it can only work with Truffle based projects with its folder structure.

I was trying to run this on my simple sol files and it was trying to find the contract folder.

### CODE REVIEW

**Comment by cgewecke** [comment]
@designium Thanks for opening this. I definitely agree the documentation could be improved here.

File: README.md [hunk 1]
```diff
@@ -15,11 +15,17 @@ find discrepancies between the coverage report and your suite's behavior, please
 + `solidity-coverage` is [Solcover](https://github.com/JoinColony/solcover)
 
 ### Install
+
+Run the following command in a Truffle based application or the root directory of your solidity project.
+
```
Comment (cgewecke) [inline README.md]: In principle you *can* run `solidity-coverage` on non-Truffle projects. The only requirement is that the instrumentation targets be located in a `contracts` folder.

It's not explicit but the installation command itself contains the necessary information about how / where to install e.g. the `--save-dev` flag says: this package should be installed in the local `node_modules` as a development dependency. 

I can see how that might not be clear to people who are unfamiliar with NPM though. Maybe this line could be simplified a little? To simply state in plain english that solidity-coverage is a local dev dependency?

File: README.md [hunk 2]
```diff
@@ -15,11 +15,17 @@ find discrepancies between the coverage report and your suite's behavior, please
...
 ```
 $ npm install --save-dev solidity-coverage
 ```
 
 ### Run
+
+The command will be run from within your application project, for each project, you will have to install solidity-coverage.
+
```
Comment (cgewecke) [inline README.md]: I think the `Run` block might benefit from a simple clarification about needing a `contracts` folder. Would you consider revising this to something like? 
```
By default solidity-coverage expects Solidity files to be located in a `contracts` folder in the root directory of your project. 
```

**Comment by cgewecke** [comment]
@designium Also wondering - what testing framework are you using, how are you running your unit tests? What file structure does that framework expect?

File: README.md (hunk 2 above)
Comment (designium) [reply README.md]: For sure.

**Comment by designium** [comment]
I am not using any unit test framework at this point, but just Truffle test. 

I believe that I would apply Jenkin as the orchestrator of all the tests I need to do later. 

But I see solidity-coverage as a tool that I can use as stand alone and part of the CI process.

File: README.md (hunk 1 above)
Comment (cgewecke) [reply README.md]: @designium I wonder if you could clarify something: you're using truffle as a test runner without using truffle's project organization. Why? How might using Jenkins require a different project pattern? 

I think it would be fantastic to have a section in the [FAQ](https://github.com/sc-forks/solidity-coverage/blob/master/docs/faq.md) addressing non-truffle installations (for example -using with Embark). Would also love to see guides to other CI setups. But a compelling use case has to be developed and they need to have a meaningful working examples at their core.

Comment (cgewecke) [reply README.md]: Another framework that would be nice to see an example installation on is [Dapp / Dapple](https://dapp.readthedocs.io/en/latest/).

Comment (designium) [reply README.md]: @cgewecke Thanks for the discussion. 

Just clarifying, I'm not using Truffle test without Truffle. I was just replying to your previous question of what test framework I'm using. 

For simple contracts, I would love to use solidity coverage that is not based on Truffle. Just simple checks. That is why I am trying to run solidity-coverage in a non-Truffle based contract.

Ideally, solidity-coverage could be run by simply:

    $solidity-coverage FILENAME.sol

or 

    $solidity-coverage /contracts/*

If that syntax works like that, without much dependencies, then it will be very easy to integrate with other testing frameworks and CI.

I understand that you are interested in including solidity-coverage in other testing frameworks.

**Comment by cgewecke** [comment]
@designium Closing for house-keeping. Some movement was made in this direction with #140 which notifies the user if there is no `contracts` folder in the cwd.
